stablehlo/reference/Ops.cpp

```diff
@@ -149,109 +166,114 @@ SmallVector<Tensor> eval(
 
   Scope scope(parent);
   scope.add(block.getArguments(), args);
-
```

Comment:
I would suggest that we keep this newline since it separates two unrelated blocks of code - creation of the scope and iteration over operations in the region.